dataprofiler/tests/profilers/test_numeric_stats_mixin_profile.py

```diff
@@ -502,3 +502,30 @@ def test_merge_num_zeros_and_negatives(self):
         num_profiler._add_helper(other1, other2)
         self.assertEqual(num_profiler.num_zeros, 0)
         self.assertEqual(num_profiler.num_negatives, 0)
+
+    def test_diff_helper(self):
+        """
+        Checks _diff_helper() works appropriately.
+        """
+        other1, other2 = TestColumn(), TestColumn()
+        other1.min = 3
+        other1.max = 4
+        other1._biased_variance = 1
+        other1.sum = 6
+        other1.match_count = 10
+        
+        other2.min = 3
+        other2.max = None
```

Comment:
does it make sense to have min=3 and max=None?

Reply:
This was just an example to show you what happens when None are involved. it might make sense if the user disables Max from options